From ddb21f4373ca2d25ea400756e22fc329c92057af Mon Sep 17 00:00:00 2001
From: Robin Gong <B38343@freescale.com>
Date: Mon, 10 Oct 2011 19:06:13 +0800
Subject: [PATCH 0491/2463] ENGR00159530-5 mc34708:modify config for new
 driver

Modify imx5_defconfig for new mc34708's core driver,new regulator driver
and new rtc driver
Signed-off-by: Robin Gong <B38343@freescale.com>
---
 arch/arm/configs/imx5_defconfig |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/arch/arm/configs/imx5_defconfig b/arch/arm/configs/imx5_defconfig
index 9c2687e..df02a3a 100755
--- a/arch/arm/configs/imx5_defconfig
+++ b/arch/arm/configs/imx5_defconfig
@@ -1249,6 +1249,8 @@ CONFIG_PMIC_DIALOG=y
 CONFIG_PMIC_DA9053Bx=y
 CONFIG_MFD_MC13783=y
 CONFIG_MFD_MC13XXX=y
+CONFIG_MFD_MC_PMIC=y
+CONFIG_MFD_MC34708=y
 # CONFIG_ABX500_CORE is not set
 # CONFIG_EZX_PCAP is not set
 # CONFIG_MFD_TPS6586X is not set
@@ -1271,6 +1273,7 @@ CONFIG_REGULATOR_FIXED_VOLTAGE=y
 CONFIG_REGULATOR_MC13XXX_CORE=y
 # CONFIG_REGULATOR_MC13783 is not set
 CONFIG_REGULATOR_MC13892=y
+CONFIG_REGULATOR_MC34708=y
 # CONFIG_REGULATOR_TPS65023 is not set
 # CONFIG_REGULATOR_TPS6507X is not set
 # CONFIG_REGULATOR_ISL6271A is not set
@@ -1961,6 +1964,7 @@ CONFIG_RTC_DRV_MXC_V2=y
 # on-CPU RTC drivers
 #
 # CONFIG_RTC_DRV_MC13XXX is not set
+CONFIG_RTC_DRV_MC34708=y
 # CONFIG_RTC_DRV_DA9052 is not set
 CONFIG_DMADEVICES=y
 # CONFIG_DMADEVICES_DEBUG is not set
-- 
1.7.7.4

